modules/viewing-the-service-logs-cli.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * osd_cluster_admin/osd_logging/osd-accessing-the-service-logs.adoc
+// * rosa_cluster_admin/rosa_logging/rosa-accessing-the-service-logs.adoc
+
+:_content-type: PROCEDURE
+[id="viewing-the-service-logs-cli_{context}"]
+= Viewing the service logs by using the CLI
+
+You can view the service logs for 
+ifdef::openshift-dedicated[]
+{product-title} 
+endif::openshift-dedicated[]
+ifdef::openshift-rosa[]
+{product-title} (ROSA) 
+endif::openshift-rosa[]
+clusters by using the {cluster-manager} CLI (`ocm`).
+
+You can view the logs for a specific cluster or for all available clusters in your Red Hat organization. You can also filter the service logs, for example by severity or by log ID.
+
+.Prerequisites
+
+* You have installed an {product-title} cluster.
+* You are the cluster owner or you have the cluster editor role.
+* You have installed and configured the latest {cluster-manager} CLI (`ocm`) on your installation host.
++
+[NOTE]
+====
+You can download the latest version of the {cluster-manager} CLI (`ocm`) on the link:https://console.redhat.com/openshift/downloads[{cluster-manager} downloads] page.
+====
+
+.Procedure
+
+. View the service logs for a cluster:
+.. List the clusters in your Red Hat organization:
++
+[source,terminal]
+----
+$ ocm list clusters
+----
++
+.Example output
+[source,terminal]
+----
+ID                                NAME            API URL                                                     OPENSHIFT_VERSION   PRODUCT ID      CLOUD_PROVIDER  REGION ID       STATE
+ifdef::openshift-dedicated[]
+1t1398ndq653vjf317a32cfjvee771dc  mycluster       https://api.mycluster.cdrj.p1.openshiftapps.com:6443        4.10.18             osd             aws             us-east-1       ready
+endif::openshift-dedicated[]
+ifdef::openshift-rosa[]
+1t1398ndq653vjf317a32cfjvee771dc  mycluster       https://api.mycluster.cdrj.p1.openshiftapps.com:6443        4.10.18             rosa            aws             us-east-1       ready
+endif::openshift-rosa[]
+----
++
+.. Obtain the external cluster ID for your cluster:
++
+[source,terminal]
+----
+$ ocm describe cluster <cluster_name> <1>
+----
+<1> Replace `<cluster_name>` with the name of your cluster.
++
+.Example output
+[source,terminal]
+----
+ID:			1t1298nhq824vjf347q12cpjvee771hc
+External ID:		f3f1a6c1-2b2b-4a55-854c-fd65e26b737b
+...
+----
++
+.. View the service logs for your cluster:
++
+[source,terminal]
+----
+$ ocm get /api/service_logs/v1/cluster_logs --parameter search="cluster_uuid = '<external_cluster_id>'" <1>
```

Comment:
Nitpick: To view the service logs for a single cluster you can also use 
$ ocm get /api/service_logs/v1/clusters/<external_cluster_id>/cluster_logs

Otherwise LGTM

Reply:
@gshilin-sdb thank you for your review and for highlighting this. For the documentation I think we only need to provide one method to achieve the same result. Is your preference to use the command that you provided instead of using the `--parameter-search` method? Thanks!

Reply:
You're right, one command is easier